sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.css

```diff
@@ -20,6 +20,12 @@
   text-shadow: none;
 }
 
+#plan-viz-graph svg g.cluster rect {
+  fill: #E5F2ff;
```

Comment:
I meant using the following style:
```
#plan-viz-graph svg g.cluster rect {
  fill: #A0DFFF;
  stroke: #3EC0FF;
  stroke-width: 1px;
}
```
<img width="259" alt="screen shot 2016-01-20 at 10 22 48 am" src="https://cloud.githubusercontent.com/assets/1000778/12458721/18ba80a0-bf60-11e5-839b-14897ac0c80c.png">

This looks more consistent with the stage page:
<img width="289" alt="screen shot 2016-01-20 at 10 24 32 am" src="https://cloud.githubusercontent.com/assets/1000778/12458733/27bb575a-bf60-11e5-9db3-e02759730255.png">

By the way, I saw a bug in the stage page:
<img width="290" alt="screen shot 2016-01-20 at 10 24 14 am" src="https://cloud.githubusercontent.com/assets/1000778/12458746/3597a2a2-bf60-11e5-9378-86914ef9a473.png">
Could you check it?